packages/editor/src/components/rich-text/index.js

```diff
@@ -407,6 +413,11 @@ export class RichText extends Component {
 	 */
 	onInput() {
 		const record = this.createRecord();
+
+		if ( ! record ) {
+			return;
```

Comment:
The input event implies that there is user input. We cannot simply ignore this input and save nothing.